content/blog/news/2023/postdam/index.en.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "CEO Summit"
+linkTitle: "CEO Summit"
+date: 2023-06-01
+---
+
+On June 1, 2023, the CEO Summit of the Memorandum of Understanding (MoU) between SNCF and Deutsche Bahn was held in Potsdam, near Berlin.
+
+Open source is one of the eight topics identified in the MoU. Loïc Hamelin, Program Director, was invited to present OSRD to JP Farandou and R Lutz, CEOs of the SNCF and DB groups.
```

Comment:
See @axrolld previous comments. We shouldn't use "JP Farandou and R Lutz", maybe "Mr. Farandou" and "Dr. Lutz"?